assets/src/domain/eventEditor/services/apollo/mutations/tickets/useReorderTickets.ts

```diff
@@ -0,0 +1,60 @@
+import { useCallback } from 'react';
+
+import { Ticket } from '../../types';
+import useReorderEntities from '../useReorderEntities';
+import { EntityTableProps } from '@appLayout/entityList';
+import { TicketsFilterStateManager as DFSM } from '@edtrServices/filterState';
+import { useTickets, useTicketQueryOptions } from '@edtrServices/apollo/queries';
+import { useUpdateTicketList } from '@edtrHooks/index';
+
+type SortResponder = EntityTableProps<Ticket, DFSM>['onSort'];
+
+interface ReorderTickets {
+	sortResponder: SortResponder;
+}
+
+const useReorderTickets = (filteredEntities: Array<Ticket>): ReorderTickets => {
+	const { sortEntities } = useReorderEntities<Ticket>({ entityType: 'TICKET' });
+	const allEntities = useTickets();
+	const queryOptions = useTicketQueryOptions();
+	const updateTicketList = useUpdateTicketList();
+
+	const updateEntityList = useCallback(
+		(updatedEntities) => {
+			updateTicketList({
+				...queryOptions,
+				data: {
+					espressoTickets: {
+						nodes: updatedEntities,
+						__typename: 'EspressoRootQueryTicketsConnection',
+					},
+				},
+			});
+		},
+		[queryOptions, updateTicketList]
+	);
+
+	const sortResponder = useCallback<SortResponder>(
+		({ destination, source }) => {
+			if (
+				!destination ||
+				(source.index === destination.index && destination.droppableId === source.droppableId) ||
+				destination.droppableId !== 'ticket-entities-table-view-droppable'
```

Comment:
you lost me here

Reply:
You can check the docs here https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/responders.md#ondragend-required

Reply:
it's for rejecting sorting that should not be handled by this callback, either because there was no change or because it was not the correct "dopzone"